src/app/base/components/node/StorageTables/AvailableStorageTable/BulkActions/BulkActions.test.tsx

```diff
@@ -183,153 +174,4 @@ describe("BulkActions", () => {
       screen.getByRole("button", { name: "Add to existing datastore" })
     ).not.toBeDisabled();
   });
-
-  it("can render the create datastore form", () => {
-    const state = rootStateFactory({
-      machine: machineStateFactory({
-        items: [
-          machineDetailsFactory({
-            system_id: "abc123",
-          }),
-        ],
-        statuses: machineStatusesFactory({
-          abc123: machineStatusFactory(),
-        }),
-      }),
-    });
-    renderWithBrowserRouter(
-      <BulkActions
-        bulkAction={BulkAction.CREATE_DATASTORE}
-        selected={[]}
-        setBulkAction={jest.fn()}
-        systemId="abc123"
-      />,
-      { state }
-    );
-
-    // Ensure correct form inputs are shown
-    expect(screen.getByRole("textbox", { name: "Name" })).toBeInTheDocument();
-    expect(screen.getByRole("textbox", { name: "Size" })).toBeInTheDocument();
-    expect(
-      screen.getByRole("textbox", { name: "Filesystem" })
-    ).toBeInTheDocument();
-    expect(
-      screen.getByRole("button", { name: "Create datastore" })
-    ).toBeInTheDocument();
-  });
-
-  it("can render the create RAID form", () => {
-    const state = rootStateFactory({
-      machine: machineStateFactory({
-        items: [
-          machineDetailsFactory({
-            system_id: "abc123",
-          }),
-        ],
-        statuses: machineStatusesFactory({
-          abc123: machineStatusFactory(),
-        }),
-      }),
-    });
-    renderWithBrowserRouter(
-      <BulkActions
-        bulkAction={BulkAction.CREATE_RAID}
-        selected={[]}
-        setBulkAction={jest.fn()}
-        systemId="abc123"
-      />,
-      { state }
-    );
-
-    // Ensure the correct form inputs are shown
-    expect(screen.getByRole("textbox", { name: "Name" })).toBeInTheDocument();
-    expect(
-      screen.getByRole("combobox", { name: "RAID level" })
-    ).toBeInTheDocument();
-    expect(screen.getByRole("textbox", { name: "Size" })).toBeInTheDocument();
-    expect(screen.getByRole("textbox", { name: "Tags" })).toBeInTheDocument();
-    expect(
-      screen.getByRole("combobox", { name: "Filesystem" })
-    ).toBeInTheDocument();
-    expect(
-      screen.getByRole("textbox", { name: "Mount point" })
-    ).toBeInTheDocument();
-    expect(
-      screen.getByRole("textbox", { name: "Mount options" })
-    ).toBeInTheDocument();
-    expect(
-      screen.getByRole("button", { name: "Create RAID" })
-    ).toBeInTheDocument();
-  });
-
-  it("can render the create volume group form", () => {
```

Comment:
I cannot find these removed tests anywhere, am I right in thinking that we have now reduced test coverage for underlying components?

I'd suggest to move these tests to appropriate places or adjust so that they fit the new flow.